conda-store-server/conda_store_server/worker/tasks.py

```diff
@@ -254,14 +255,20 @@ def task_delete_build(self, build_id):
 
         build = api.get_build(db, build_id)
 
+        # Deletes build artifacts for this build
         conda_store.log.info(f"deleting artifacts for build={build.id}")
         for build_artifact in api.list_build_artifacts(
             db,
             build_id=build_id,
             excluded_artifact_types=settings.build_artifacts_kept_on_deletion,
         ).all():
             delete_build_artifact(db, conda_store, build_artifact)
-        conda_store.db.commit()
+
+        # Updates build size and marks build as deleted
+        build.deleted_on = datetime.datetime.utcnow()
+        build.size = 0
+
+        db.commit()
```

Comment:
Bug fix: `CondaStore` object has no attribute `db`, need to just use `db` for this